rpm/archivematica-storage-service/etc/archivematica-storage-service.env

```diff
@@ -14,3 +14,5 @@ DJANGO_ALLOWED_HOSTS=*
 SS_GUNICORN_BIND=127.0.0.1:7500
 
 PYTHONPATH=/usr/lib/archivematica/storage-service
+
+REQUEST_CA_BUNDLE=/etc/pki/tls/certs/ca-bundle.crt
```

Comment:
This was accidentally added to the duplicate (`rpm/archivematica/etc/archivematica-storage-service.env`).